static/src/stylesheets/module/content/_live-blog.scss

```diff
@@ -271,6 +271,25 @@ $block-padding-right: $gs-gutter;
 
     .block-elements {
 
+        /* Bylines
+           ========================================================= */
+        .liveblog-block-byline {
+            border-bottom: 1px dotted #dcdcdc;
```

Comment:
`neutral-4` - sorry, missed this one earlier. basically, the only colour you should spec manually is `#ffffff`